valdr-ng/projects/valdr-ng-showcase/src/app/valdr-form/valdr-form.component.spec.ts

```diff
@@ -1,16 +1,21 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { ValdrFormComponent } from './valdr-form.component';
+import {ValdrNgModule, ValdrNgService} from '../../../../valdr-ng/src/public-api';
```

Comment:
formatting seems a bit off throughout the PR

Reply:
Good point: https://github.com/Stanoja/ValdrNG/pull/9